packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -6,6 +6,74 @@ import 'package:flutter/widgets.dart';
 import 'bottom_tab_bar.dart';
 import 'theme.dart';
 
+/// Coordinates tab selection between a [CupertinoTabBar] and a [CupertinoTabScaffold].
+///
+/// The [currentIndex] property is the index of the selected tab, changing its value updates
+/// the actively displayed tab of the [CupertinoTabScaffold] the [CupertinoTabController] controls,
+/// as well as the currently selected tab item of its [CupertinoTabBar].
+///
+/// There're 2 ways to programmatically change the actively displayed tab of a [CupertinoTabScaffold]
```

Comment:
I wouldn't include this section. It does work but it would break as soon as we start animating the tab transitions etc.